src/dist/replication/lib/replica_bulk_load.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#include <fstream>
+#include <dsn/dist/block_service.h>
+#include <dsn/dist/fmt_logging.h>
+#include <dsn/dist/replication/replication_app_base.h>
+#include <dsn/utility/filesystem.h>
```

Comment:
Include headers in the following order: Related header, C system headers, C++ standard library headers, other libraries' headers, your project's headers.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes